cli/utilities/utils.py

```diff
@@ -520,3 +520,40 @@ def get_disk_devlinks(node, disk):
         elif "by-path" in path:
             by_path.append(path)
     return by_id, by_path
+
+
+def podman_login(node, registry, username, password):
```

Comment:
For podman logging, method already available - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L15